src/index.html

```diff
@@ -17,6 +17,77 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a
+        href="index.html"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Logo"
+        />
```

Comment:
Add empty lines between multiline sibling blocks of HTML to add some 'air' and simplify reading. For example, add an empty line before the <nav> element on line 30.

Reply:
Add empty line before the element on line 30.